server/fishtest/templates/tests_stats.mak

```diff
@@ -54,6 +54,16 @@
   if has_pentanomial:
       results5 = run["results"]["pentanomial"]
       results5_ = fishtest.stats.LLRcalc.regularize(results5)
+      results5_pair_ratio = (
+          sum(results5[3:]) / sum(results5[0:2])
+          if sum(results5[0:2]) > 0
+          else float("nan")
+      )
+      results5_point_ratio = (
+          (2 * results5[4] + results5[3]) / (2 * results5[0] + results5[1])
+          if sum(results5[0:2])
```

Comment:
can this be a division by 0?

Reply:
No, I used the ternary operator to set "nan", let me write on a single line:
```python
a = b / c if c else float("nan")
```

Reply:
yeah, misread the code... the end index is not part of the array slice in python.

Reply:
In that line I lost the `> 0` in a pasto, worked for luck, I improved the conditional statement with the shorter/faster `any()`.

It's possible to use also `float("+inf")`